_ingest-pipelines/processors/dot-expander.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: default
+title: Dot expander
+parent: Ingest processors
+nav_order: 65
+---
+
+# Dot expander 
+
+The `dot_expander` processor is a tool that helps you work with hierarchical data. It transforms fields containing dots into object fields, making them accessible to other processors in the pipeline. Without this transformation, fields with dots cannot be processed.
+
+The following is the syntax for the `dot_expander` processor:
+
+```json
+{
+  "dot_expander": {
+    "field": "field.to.expand" 
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `dot_expander` processor.
+
+Parameter | Required/Optional | Description |
+|-----------|-----------|-----------|
+`field`  | Required  | The field to expand into an object field. |
+`path` | Optional | The field is only required if the field to be expanded is nested within another object field. This is because the `field` parameter only recognizes leaf fields. |
```

Comment:
```suggestion
`path` | Optional | This field is only required if the field to be expanded is nested within another object field. This is because the `field` parameter only recognizes leaf fields. |
```